source/includes/_api-reference.md

```diff
@@ -50,18 +58,52 @@ request.AddHeader("Authorization", "Basic [insert credentials here]");
 var request = new RestRequest(Method.GET);
 
 request.AddHeader("Authorization", "ApiKey [insert key here]");
+
+
+// Access Token Authentication
+var request = new RestRequest(Method.GET);
+ 
+request.AddHeader("Authorization", "Bearer [insert Access token here]");
 ```
 
-Intelex utilizes two methods for authentication to the API- Basic and ApiKey.
+Intelex utilizes three methods for authentication to the API- Basic, ApiKey and Secure.
 
 Basic authentication is performed via HTTP Basic Auth and your Intelex user credentials (Username:Password). The credentials are then verified by Intelex and access will be granted or denied accordingly.
 
 ApiKey authentication can be performed by generating API access key(s) from the Intelex User Profile menu (note that this page is only accessible to full-access and admins on the site). 
 The unique generated key can then be used to access the Intelex REST API and used to authenticate and prove that the user is who they claim to be.
 
+Intelex V6 API uses secure token based authentication. Only V6 Admin users can access the API.
```

Comment:
"Only V6 Admin users can access the API." - this is wrong, our recommendation is to create a dedicated user not admin to use for the integration.

Reply:
Updated the sentence as "Only dedicated users can access the API."